resources/keb/templates/deployment.yaml

```diff
@@ -67,8 +67,6 @@ spec:
               value: "{{ .Values.exposeSchemaWithRegionRequired }}"
             - name: APP_BROKER_REGION_PARAMETER_IS_REQUIRED
               value: "{{ .Values.regionParameterIsRequired }}"
-            - name: APP_BROKER_ALLOW_NETWORKING_PARAMETERS
```

Comment:
This should be removed after KEB release with changes from this PR. After the release in the PR with the image bump, this can be removed. The reason is that the chart will be propagated without the env, but there will be an old KEB image expecting the `APP_BROKER_ALLOW_NETWORKING_PARAMETERS` flag.